### PR TITLE
Saint Roumain Militia Content Expansion Part 1: The Prelude

### PR DESCRIPTION
**Pull request by Imaginos16**
## About The Pull Request
This PR is a start to an upcoming project to give the SRM some more love.

This adds several weapons locked behind admin spawn as purchasable items in the outpost console.

- The Contender for 2250 Credits (And 45-70 ammunition boxes for 500 Credits)
- The Scout for 5500 Credits (And .300 Magnum boxes for 500 Credits, with stripper clips for 750 Credits)
- The Double Barrel Shotgun for 1000 Credits (Cheaper than Hellfire and Brimstone)
- The Firebrand for 1250 Credits (And .357 Ammo Boxes for 250 Credits with specialty editions for 500 Credits)
- The .38 Derringer for 350 Credits

This PR also granularizes disposable guns and makes them 150 credits per unit.

Finally, this PR removes the Scarborough Revolver as a purchasable weapon in the outpost store, being now replaced by the Firebrand. Gorlex and Syndicate captains still have it for the foreseeable future!

## Why It's Good For The Game
Considering how a lot of content the SRM has is currently locked in Admin Hell, hopefully adding these weapons to the outpost will help give more character for a potential "indie armory" in outposts, as well as giving the SRM more options for fights without having to beg admins to use them. The Ashhand, Vickland, Huntsman and Firestorm are currently locked behind admin spawn for a future SRM ERT.

I swear to fucking god after I'm done with this entire rework I still get no people joining my ships I'm gonna go nuclear.

## Changelog

:cl: PositiveEntropy
add: The HP Firebrand, HP Scout, HP Contender, .38 Derringer and Double Barrel Shotgun are now purchasable in the outpost store.
add: .357 Ammo Boxes, .300 Magnum Stripper Clips and Ammo Boxes are now implemented and purchasable in the outpost.
add: .45-70 Ammo Boxes are now purchasable in the outpost.
fix: The Disposable Gun crate is now granularized.
del: The Scarborough Revolver is no longer purchasable in the outpost store.
/:cl:

### CODE REVIEW

**Review by rye-rice — APPROVED** [review]
sprites made by me

**Comment by meemofcourse** [comment]
> I swear to fucking god after I'm done with this entire rework I still get no people joining my ships I'm gonna go nuclear.

:clueless:

**Comment by Martinpachu** [comment]
that's awesome? kind of bodacious.

**Comment by goober3** [comment]
> :clueless:

6e259c58dfa31b337ec6954f63a22dc5

File: code/modules/cargo/packs/gun.dm [hunk 1]
```diff
@@ -36,15 +39,15 @@
 	cost = 1000
 	contains = list(/obj/item/gun/ballistic/automatic/pistol/candor)
 
-/datum/supply_pack/gun/revolver
-	name = "Scarborough Revolver Crate"
-	desc = "Contains a concealable Scarborough revolver, chambered in .357."
+/datum/supply_pack/gun/pepperbox
+	name = "HP Firebrand Pepperbox Revolver Crate"
+	desc = "Contains a concealable pepperbox revolver manufactured by the Saint Roumain Militia, chambered in .357."
```
Comment (Apogee-dev) [inline code/modules/cargo/packs/gun.dm:44]: Minor terminology gripe here, it’s a “pepperbox pistol,” not a pepperbox revolver. I know they’re sometimes called that, but it’s a little less confusing to not call it a revolver imo

I’m also aware the obj itself as well as related objects refer to it as such :shrivel:

Comment (Apogee-dev) [reply code/modules/cargo/packs/gun.dm:44]: Yknow honestly this isn’t your problem to deal with, ignore it. Maybe if it keeps bugging me I’ll make a pedantic terminology fix PR

**Review by meemofcourse — APPROVED** [review]
WOW!!! MORE THINGS TO SHOOT (with)!!!

**Pull request merged**